### ISSUE TITLE
Brpoplpush::RedisScript::LuaError: WRONGTYPE Operation against a key holding the wrong kind of value

### ISSUE THREAD

**Issue by yaoyi** [mhenrixon/sidekiq-unique-jobs#491]
current implementation in `find_digest_in_queues.lua`
```lua
local pagination  = redis.call("SCAN", cursor, "MATCH", "*queue:*", "COUNT", count)
```
we have a key `cron_job:scheduler_create_sqs_queue:enqueued`, it is not a queue, but will match "\*queue\*", which cause the error above, guess either fix this case or warn user in the readme

**Comment by mhenrixon** [comment]
Shouldn’t be too hard to fix. I doubt the leading asterisk is necessary

**Comment by mhenrixon** [comment]
This one was already addressed with #493 so closing this for now. I'll be releasing a new version today.

**Comment by mhenrixon** [comment]
Released as v7.0.15
